ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/PivotSelectorFromPeers.java

```diff
@@ -58,15 +55,19 @@ public Optional<FastSyncState> selectNewPivotBlock() {
 
   @Override
   public CompletableFuture<Void> prepareRetry() {
+    final long estimatedPivotBlock = conservativelyEstimatedPivotBlock();
     final TrailingPeerLimiter trailingPeerLimiter =
         new TrailingPeerLimiter(
             ethContext.getEthPeers(),
             () ->
                 new TrailingPeerRequirements(
-                    conservativelyEstimatedPivotBlock(), syncConfig.getMaxTrailingPeers()));
+                    estimatedPivotBlock, syncConfig.getMaxTrailingPeers()));
     trailingPeerLimiter.enforceTrailingPeerLimit();
 
-    return waitForPeers(syncConfig.getSyncMinimumPeerCount());
+    return ethContext
+        .getEthPeers()
+        .waitForPeer((peer) -> peer.chainState().getEstimatedHeight() >= estimatedPivotBlock)
+        .thenRun(() -> {});
```

Comment:
This is the only major change in behavior. I couldn't figure out why on earth we would wait for more than one peer when we could just as easily get started with the first peer that's high enough to contain the pivot block we're looking for. Let me know what you think.

Reply:
I think that in PoW we were originally picking a pivot block that had to be validated by 5 peers. Has that changed? Have we made that configurable?

Reply:
Yeah we need that for POW, thought was configurable as sometimes we tell people to lower that if they are having trouble syncing.

We should keep the logic the same here so it still works with POW for now. With changes to syncing on POS we won't even use the PivotSelectorFromPeers.

Reply:
Good news is, it'll still wait for the configured number of peers to confirm the pivot block. It'll just retry a few times until the peers are present.